diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/build.gradle.kts b/node_modules/expo-modules-core/expo-module-gradle-plugin/build.gradle.kts
index 9697cea..40a970e 100644
--- a/node_modules/expo-modules-core/expo-module-gradle-plugin/build.gradle.kts
+++ b/node_modules/expo-modules-core/expo-module-gradle-plugin/build.gradle.kts
@@ -2,7 +2,7 @@ import org.gradle.api.tasks.testing.logging.TestExceptionFormat
 import org.jetbrains.kotlin.gradle.tasks.KotlinCompile

 plugins {
-  kotlin("jvm") version "2.1.20"
+  kotlin("jvm") version "1.9.24"
   id("java-gradle-plugin")
 }

diff --git a/node_modules/expo-modules-core/expo-module-gradle-plugin/settings.gradle.kts b/node_modules/expo-modules-core/expo-module-gradle-plugin/settings.gradle.kts
new file mode 100644
index 0000000..a400c3b
--- /dev/null
+++ b/node_modules/expo-modules-core/expo-module-gradle-plugin/settings.gradle.kts
@@ -0,0 +1 @@
+rootProject.name = "expo-module-gradle-plugin"
